index.html

```diff
@@ -0,0 +1,310 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/fontawesome.min.css">
+    <title>Francis</title>
+</head>
+<body>
+    <main id="main-container"   >
+        <!-- Top navigation section -->
+        <div id="nav-headline-background">
+            <header id="top-navigation">
+                <a id="logo">
+                    <p id="logo-text">Welcome</p>
```

Comment:
- [ ] [OPTIONAL] I think it would be better to use your name as the logo instead of "Welcome" 😉